src/config/dynamic-theme-fixes.config

```diff
@@ -7454,6 +7443,27 @@ a[href*="about/products"]
 img[src$="menu_black_24dp.png"]
 #assistive-chips .e2moi img
 button[guidedhelpid="searchbutton"]
+a[href^="//myaccount.google.com/yourdata/maps"] > div
+img[src^="//www.google.com/images/branding/lockups/"][alt="Google Maps"]
+div[role="checkbox"] > div
+div[aria-label="Toggle star"][role="checkbox"]
+#legendPanel > div > div > div > div > div > div > div > div > div[jsaction^="keydown"][jsshadow=""][role="checkbox"][style=""]
+#legendPanel > div > div > div > div > div > div > div > div > div > div > div > div[jsaction^="keydown"][jsshadow=""][role="checkbox"]
+.QA0Szd-jcJzye-l4eHX
+.QA0Szd-ibnC6b-Bz112c
+.QA0Szd-ibnC6b-qwU8Me-Bz112c
+div[class="nJjxad-bMcfAe-haAclf nJjxad-HzV7m"] > div > div
```

Comment:
Please try to avoid such selectors, they have a low-guarantee that they will still work over a few months.